costmanagement-metrics-operator/3.3.1-test-1/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM scratch
```

Comment:
This directory is wrong. It should just be `3.3.1`, not `3.3.1-test-1`

Reply:
Oof, I should use a diff terminal tab for testing things. 
Thanks for spotting this one!